src/ooxml/java/org/apache/poi/xssf/usermodel/XSSFCell.java

```diff
@@ -53,6 +53,7 @@ Licensed to the Apache Software Foundation (ASF) under one or more
 import org.openxmlformats.schemas.spreadsheetml.x2006.main.CTCellFormula;
 import org.openxmlformats.schemas.spreadsheetml.x2006.main.STCellFormulaType;
 import org.openxmlformats.schemas.spreadsheetml.x2006.main.STCellType;
+import org.apache.poi.xssf.model.CalculationChain;
```

Comment:
could you review the order in which you add imports?

Reply:
ok